diff -r -U 3 a/alex.cabal b/alex.cabal
--- a/alex.cabal	1970-01-01 01:00:01.000000000 +0100
+++ b/alex.cabal	2011-03-14 00:08:54.000000000 +0100
@@ -68,7 +68,7 @@
   if flag(small_base)
     build-depends: base >= 2.1, array, containers, directory
   else
-    build-depends: base >= 1.0
+    build-depends: base >= 1.0 && < 2.1, directory
 
   build-depends: base < 5
 
Only in b: alex.cabal.orig
Only in b/dist: alex-2.3.5.tar.gz
Only in b: Setup
Only in b: Setup.hi
Only in b: Setup.o
