diff -U 3 a/berkeleydb.cabal b/berkeleydb.cabal
--- a/berkeleydb.cabal	1970-01-01 01:00:01.000000000 +0100
+++ b/berkeleydb.cabal	2009-11-28 02:52:02.000000000 +0100
@@ -4,7 +4,6 @@
 Maintainer:      David Himmelstrup <lemmih@gmail.com>
 Copyright:       2008 David Himmelstrup <lemmih@gmail.com>
 Build-Type:      Simple
-Build-Depends:   base, bytestring, binary
 Exposed-Modules: Data.BerkeleyDB
 Other-Modules:   Data.BerkeleyDB.Internal
                  Data.BerkeleyDB.IO
@@ -29,3 +28,12 @@
   and a very small memory footprint.
   .
   Tested with libdb4.6
+
+flag split-base
+
+Library
+  Build-Depends:   bytestring, binary
+  if flag(split-base)
+    build-depends:	base >= 3 && < 5, syb
+  else
+    build-depends:	base < 3
