Common subdirectories: a/Database and b/Database
diff -U 3 a/haskelldb-hdbc-odbc.cabal b/haskelldb-hdbc-odbc.cabal
--- a/haskelldb-hdbc-odbc.cabal	1970-01-01 01:00:01.000000000 +0100
+++ b/haskelldb-hdbc-odbc.cabal	2009-11-28 04:53:41.000000000 +0100
@@ -14,8 +14,8 @@
     mtl, 
     haskelldb>=0.12,
     haskelldb-hdbc>=0.12,
-    HDBC>=2.0.0 && < 2.2.0,
-    HDBC-odbc>=2.0.0 && < 2.2.0
+    HDBC>=2.0.0 && <= 2.2.1,
+    HDBC-odbc>=2.0.0 && <= 2.2.1
   Exposed-Modules:
         Database.HaskellDB.HDBC.ODBC
 
