diff -U 3 a/haskeline.cabal b/haskeline.cabal
--- a/haskeline.cabal	1970-01-01 01:00:01.000000000 +0100
+++ b/haskeline.cabal	2009-11-28 19:31:12.000000000 +0100
@@ -38,7 +38,7 @@
         cpp-options:    -DOLD_BASE
     }
     else {
-        if impl(ghc>=6.11) {
+        if impl(ghc>=6.10.4) {
             Build-depends: base >=4.1 && < 4.3, containers>=0.1 && < 0.4, directory==1.0.*,
                            bytestring==0.9.*
         }
