Common subdirectories: a/Database and b/Database
diff -U 3 a/haskelldb-hdbc-sqlite3.cabal b/haskelldb-hdbc-sqlite3.cabal
--- a/haskelldb-hdbc-sqlite3.cabal	1970-01-01 01:00:01.000000000 +0100
+++ b/haskelldb-hdbc-sqlite3.cabal	2009-11-28 04:54:32.000000000 +0100
@@ -14,7 +14,7 @@
   Build-depends: 
     mtl, 
     haskelldb>=0.12, haskelldb-hdbc>=0.12,
-    HDBC>=2.0.0 && < 2.2.0, HDBC-sqlite3>=2.0.0 && < 2.2.0
+    HDBC>=2.0.0 && <= 2.2.1, HDBC-sqlite3>=2.0.0 && <= 2.2.1
   if flag(split-base)
     Build-depends: base >= 3.0
   else
