diff -r -U 3 a/hsql-mysql.cabal b/hsql-mysql.cabal
--- a/hsql-mysql.cabal	1970-01-01 01:00:01.000000000 +0100
+++ b/hsql-mysql.cabal	2010-05-18 16:55:19.000000000 +0200
@@ -8,15 +8,15 @@
 ghc-options: -O2 
 build-depends: base >=4 && < 5, hsql >= 1.8, Cabal 
 extensions: ForeignFunctionInterface, CPP 
-include-dirs: Database/HSQL, /usr/include/mysql 
+include-dirs: Database/HSQL
 build-type: Simple 
 extra-source-files: Database/HSQL/HsMySQL.h 
 extra-libraries: mysqlclient 
-extra-lib-dirs: /usr/lib,/usr/lib/mysql 
+extra-lib-dirs:
 exposed-modules: 
   Database.HSQL.MySQL
   DB.HSQL.MySQL.Type
   DB.HSQL.MySQL.Functions
 maintainer: Chris Done <chrisdone@gmail.com>
 license-file: LICENSE
-cabal-version: >= 1.6
\ No newline at end of file
+cabal-version: >= 1.6
