diff -r -U 3 a/QuickCheck.cabal b/QuickCheck.cabal
--- a/QuickCheck.cabal	1970-01-01 01:00:01.000000000 +0100
+++ b/QuickCheck.cabal	2010-01-04 06:36:08.000000000 +0100
@@ -21,6 +21,7 @@
 cabal-version: >=1.2
 
 flag split-base
+flag base4
 
 library
   exposed-modules:
@@ -33,7 +34,12 @@
           Test.QuickCheck.Utils,
           Test.QuickCheck
   if flag(split-base)
-    build-depends:	base >= 3 && < 4, random
+    build-depends:	base >= 3 && < 5, random
+    if flag(base4)
+      build-depends: base >= 4
+      cpp-options: -DBASE4
+    else
+      base < 4
   else
     build-depends:	base < 3
   extensions:	CPP
